ui/apps/platform/cypress/helpers/networkGraphPF.js

```diff
@@ -20,7 +20,7 @@ const routeMatcherMapForClusterInNetworkGraph = {
     },
 };
 
-const clusterNamespacesTarget = '/v1/sac/clusters/*/namespaces';
+const clusterNamespacesTarget = '/v1/sac/clusters/*/namespaces?permissions=NetworkGraph&permissions=Deployment';
```

Comment:
@rhybrillou See if this pattern fixes the problem when request has query parameters:

https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/cypress/helpers/policies.js#L87-L92